test/framework/framework.go

```diff
@@ -91,6 +92,23 @@ func (t *TestFramework) MustTempFile(pattern string) string {
 	return path.Name()
 }
 
+// EnsureRuntimeDeps creates a directory which contains faked runtime
+// dependencies for the tests.
+func (t *TestFramework) EnsureRuntimeDeps() string {
+	dir := t.MustTempDir("crio-testunig-default-runtime-")
+	for dep, content := range map[string]string{
+		"crun":    "",
+		"conmon":  "#!/bin/sh\necho 'conmon version 2.1.12'",
+		"nsenter": "",
+	} {
+		runtimeDep := filepath.Join(dir, dep)
+		gomega.Expect(os.WriteFile(runtimeDep, []byte(content), 0o755)).
+			NotTo(gomega.HaveOccurred())
```

Comment:
```suggestion
		gomega.Expect(os.WriteFile(runtimeDep, []byte(content), 0o755)).NotTo(gomega.HaveOccurred())
```

I am not sure where this break-on-dot comes from, but this is not Go style.

Reply:
It's the valid syntax to break long function chains. :shrug:

Reply:
Fair enough. You can tell I am not a fan of this approach. :smile: 

_For posterity:_

This works because of the implicit semicolon insertion rules. The same reason why this...

```go
Object
  .A()
  .B()
  .C()
```

would not work, but...

```go
Object.
  A().
  B().
  C()
```

works. The same way the following works:

```go
	for
	i := 0
	i < 10
	i++ {
		// Some code here.
	}
```

But we tend not do that.